### ISSUE TITLE
Use a single option to turn on Spotlight

### ISSUE THREAD

**Issue by rdmark** [Netatalk/netatalk#1618]
Right now we have three afp.conf options that all need to be turned on to enable Spotlight.

- spotlight
- start tracker
- start dbus

The two latter were added in https://sourceforge.net/p/netatalk/feature-requests/91/ with little context. I can't immediately think of a reason why you'd ever enable spotlight with also wanting to have tracker and dbus running. Maybe there was some corner case bug or instability early on when spotlight functionality was fresh?

Anyhow, I think these three should be compressed into a single option.

**Comment by singular0** [comment]
I think this split has something to do with #54 (i.e. if you run indexer as a separate service on system dbus, it starts to make sense).

**Comment by rdmark** [comment]
That's a good point. So there may be a case for controlling the indexer dbus separately.

Can you see a reason for keeping spotlight and tracker (indexer) as separate options? The only reason I can think of would be if we in the future introduced support for multiple indexing backends, and controlled the active backend dynamically rather than at compile time. E.g. https://github.com/Netatalk/netatalk/issues/1409

But at that point the option shouldn't be called `start tracker` but rather something like `indexer backend`.

**Comment by singular0** [comment]
No, at the moment I don't see any scenario in which we may want to enable spotlight without controlling indexer process.

**Comment by rdmark** [comment]
After taking another look at the code today, I'm leaning towards removing `start dbus` too, for now. It really does not serve any practical purpose in the current implementation, and only adds to the clutter and complexity of netatalk configuration.
